pkg/splunk/enterprise/configuration.go

```diff
@@ -411,13 +412,15 @@ splunk:
     hec_enableSSL: 0
     hec_token: "%s"
     password: "%s"
+    pass4SymmKey: "%s"
     idc:
         secret: "%s"
```

Comment:
I think idc needs this too?

Reply:
I was leaving it as `splunk.idc.secret` so that it works with both old and new containers (backwards compat).